ptx/sec_numerical_integration.ptx

```diff
@@ -63,7 +63,12 @@
         <caption/>
       <!-- START figures/fig_numerical1a.tex -->
         <image xml:id="img_numerical1a">
-          <description></description>
+        <shortdescription>integral of e^{-x^2} from 0 to 1</shortdescription>
+          <description>
+          <p> This is a graph of <p>f(x)=e^{-x^2}</p>.starting at <m>(0,1)</m> the line slopes down towards the x-axis. The line goes from <m>(0,1)</m> to close to <m>(1,.75)<m>. We consider the area between <m>(0,1)</m>
+         ,<m>(1,.75)</m>,<m>(1,0)</m> and <m>(0,0)</m>.
```

Comment:
Note the leading comma. The comma should be attached at the end of each point, not the beginning.
And there should be a space after each comma.